pyprep/prep_pipeline.py

```diff
@@ -61,8 +61,12 @@ class PrepPipeline:
     raw_non_eeg : mne.raw | None
         The non-eeg part of the data. It is not processed when calling
         the fit method. If the input was only EEG it will be None.
-    noisy_channels_original : list
-        bad channels before robust reference
+    noisy_channels_original : dictionary
+        detailed bad channels in each criteria before robust reference
+    noisy_channels_before_interpolation : dictionary
+        detailed bad channels in each criteria just before interpolation
+    noisy_channels_after_interpolation : dictionary
+        detailed bad channels in each criteria just after interpolation
```

Comment:
In the numpydoc docstr we try to put the datatypes on the right side of the colon `:`, see this:

```python
>>> mydict = {"a": 1}
>>> type(mydict)  # this is not going to print "dictionary"
dict
```

We are probably doing this wrong also for the `mne.raw` part, but we don't have to fix it now.

Also: Capitalization and ending each param description with a punctuation mark would be good style (I think).

```suggestion
    noisy_channels_original : dict
       Detailed bad channels in each criteria before robust reference.
    noisy_channels_before_interpolation : dict
        Detailed bad channels in each criteria just before interpolation.
    noisy_channels_after_interpolation : dict
        Detailed bad channels in each criteria just after interpolation.
```